src/app/carbon-estimation/carbon-estimation.constants.ts

```diff
@@ -1,10 +1,14 @@
+import { TableItem } from '../carbon-estimation-table/carbon-estimation-table.component';
 import { ChartOptions } from '../types/carbon-estimator';
 
 export enum EmissionsColours {
   Upstream = '#40798C',
+  UpstreamLight = '#bfdae2',
   Direct = '#CB3775',
   Indirect = '#91234C',
+  OperationLight = '#f2afd1',
   Downstream = '#4B7E56',
+  DownstreamLight = '#c1d9c3',
```

Comment:
```suggestion
  Upstream = '#40798C',
  UpstreamLight = '#BFDAE2',
  Direct = '#CB3775',
  Indirect = '#91234C',
  OperationLight = '#F2AFD1',
  Downstream = '#4B7E56',
  DownstreamLight = '#C1D9C3',
```
Very minor nitpick but can we match the case in the new hex codes with the pre-existing ones?